src/lbcamden/components/search/template.njk

```diff
@@ -2,15 +2,45 @@
   {% set classNames = classNames + " " + params.classes %}
 {% endif %}
 
-<form class="lbcamden-search {{ classNames }} {% if params.alt %} lbcamden-search--alt{% endif %}" action="/"
-      method="post" autocomplete="off" role="search">
-  <label class="govuk-label govuk-label--l govuk-visually-hidden" for="lbcamden-search__box">
-    {{ params.labelText | default('Search Camden.gov.uk') }}
+{%- if params.action %} 
+  {% set action = params.action %}
+  {% else %} 
+    {% set action = 'search' %}
+{% endif %}
+
+{%- if params.postMethod %}
+  {% set method = 'post' %}
```

Comment:
Maybe just let people specify `method` as a string? method="get", method="post", etc. Defaulting to get is right, though.